inc/compat/themes/compat-theme-minimog.php

```diff
@@ -22,6 +22,16 @@ public function hooks() {
 		// Scripts
 		add_action( 'wp_enqueue_scripts', array( $this, 'deregister_woocommerce_scripts' ), 20 );
 		add_action( 'wp_enqueue_scripts', array( FluidCheckout_Enqueue::instance(), 'replace_woocommerce_scripts' ), 20 );
+
+		// Remove checkout payment info heading
+		add_filter( 'fc_content_section_class', array( $this, 'remove_checkout_payment_heading' ), 20 );
```

Comment:
Not using the correct hook. For consistency, all hooks need to be added in the functions `hooks()`, `late_hooks()` or `very_late_hooks()`.

The hook `fc_content_section_class ` used is way too late for adding or removing hooks. This might work because the content section will probably be output before the payment section, however, for other type of hooks this will be a problem.

If removing the hook `woocommerce_checkout_after_order_review ` (LN34) does not work directly in the `hooks()` function, try using the other two functions.

Reply:
As it was working directly in hooks i thaught it would be less efficient to put it in late_hooks without a good reason, should I still put it inside late_hooks or very_late_hooks in this case? it sounds like bloating

Reply:
I'm assuming you mean it "did **not** work directly in `hooks()`"...

The reason we have `late_hooks()` and `very_late_hooks` is because some plugins and themes will register their hooks too late for the `hooks()` function to be able to make changes.

The `late_hooks()` and `very_late_hooks` functions run before any HTML is generated. By using these functions we can make sure all Fluid Checkout hooks are added before any HTML is generated, and that these hooks will also be available for AJAX requests.

We can only add or change hooks after the `very_late_hooks()` in very special cases, usually when other plugins and themes do not follow this rule of having all hooks added and ready before generating HTML and therefore we need to "break" this rule to be able to do what we need to do.

Remember that `late_hooks()` is always tied to the `init` hook, and `very_late_hooks` is always tied to the `wp` hook at priority `100`:

```php
// Late hooks
add_action( 'init', array( $this, 'late_hooks' ), 100 );
		
// Very late hooks
add_action( 'wp', array( $this, 'very_late_hooks' ), 100 );
```

See the file `inc/checkout-steps.php` in the Fluid Checkout Lite project.